docs/search_config.md

```diff
@@ -0,0 +1,139 @@
+- **Calibration with Larger Values:** For full-scale calibration, increase both population_size and generations to achieve a more comprehensive search. We recommend population sizes between 30–100 and similar ranges for generations. Note that the effectiveness of larger population sizes or more generations depends on the number and granularity of available calibration adjustment knobs. If only a few knobs are being tuned, excessively large values may not provide meaningful improvements.
+- **Balance Exploration and Intensification:**
+    - if the search space is complex, consider increasing the crossover and mutation probabilities to ensure thorough exploration and solution coverage.
+    - Higher crossover and mutation probabilities (greater than 0.1 or 10%) can enhance the algorithm's ability to investigate new search spaces, while lower probabilities tend to concentrate the search on refining and intensifying existing solutions.
+    - Higher crossover and mutation probabilities may result in longer search time.
+- **Evaluating Results:** Experiment with different parameter settings and compare outcomes to achieve an optimal balance between solution quality and computational efficiency.
+- **Tune Based on Constraints:** When computational resources are limited, smaller populations, fewer generations, and smaller crossover and mutation probabilities may still yield useful results. It’s often necessary to experiment with different values to find the optimal values for a particular problem.
+
+## Value Choices
+
+These arrays contain the discrete options that the genetic algorithm will evaluate to calibrate the model. The values are multipliers of the element values in the hpxml model. 1 means the exact value that is in the original model. 0.1 means 1/10th of the original value. 10 means 10x the original value. Each home should have a custom config file for what is known about that home in particular. For instance, if a blower door test was completed on a home, you could probably have very small or no range for the air leakage choices in that home. We think a safe principle is to start with narrow ranges and few options the first time you attempt calibration of a home.
+
+- Numbers farther away from 1 signify more uncertainty in the original model.
+    - If those settings aren't able to generate a model that meets the acceptance criteria, try expanding the range.
+        - Instead of `[0.5, 1, 2]`, consider `[0.25, 1, 4]`
+- More values inside the array give the genetic algorithm more granularity to find a solution.
+    - If the quality of the output isn't very good, more granularity could help.
+        - Instead of `[0.5, 1, 2]`, consider `[0.5, 0.75, 1, 1.5, 2]`
+
+### The choices in the config file adjust the following hpxml values
+
+If your model doesn't contain any of them, that field is ignored.
```

Comment:
Is that right? I thought it will use the choices from the default config instead (which a user should _not_ rely on).

Reply:
Perhaps I didn't word this very well. I mean that if the hpxml model doesn't have, for instance, a permanent spa, then values in the `misc_load_multiplier_choices` array won't create a new one, nor will they cause the modification of the model to fail.

It is true that if a custom config file (for a specific home) does not contain any given field (in the whole config file, not just the value choices), it will take the value of the default config file.